website/fiddle/package.json

```
@@ -22,15 +22,15 @@
     "dev": "node patch.js && node build_patch_ui5.js && webpack-dev-server  --content-base public"
   },
   "dependencies": {
-    "@luigi-project/client": "1.22.0",
-    "@luigi-project/core": "1.22.0",
-    "@luigi-project/plugin-auth-oauth2": "1.22.0",
-    "@luigi-project/plugin-auth-oidc": "1.22.0",
+    "@luigi-project/client": "1.23.1",
+    "@luigi-project/core": "1.23.1",
+    "@luigi-project/plugin-auth-oauth2": "1.23.1",
+    "@luigi-project/plugin-auth-oidc": "1.23.1",
```

```suggestion
    "@luigi-project/client": "1.24.0",
    "@luigi-project/core": "1.24.0",
    "@luigi-project/plugin-auth-oauth2": "1.24.0",
    "@luigi-project/plugin-auth-oidc": "1.24.0",
```
We can update to the latest now.